Quadriga/src/main/java/edu/asu/spring/quadriga/accesschecks/impl/ProjectSecurityChecker.java

```diff
@@ -85,41 +82,48 @@ public boolean isProjectOwner(String userName, String projectId)
 
     @Override
     @Transactional
-    public boolean ownsAtLeastOneProject(String userName)
-            throws QuadrigaStorageException {
+    public boolean ownsAtLeastOneProject(String userName) throws QuadrigaStorageException {
         // check if the use is associated with any project
         return accessManager.getNrOfOwnedProjects(userName) > 0;
     }
 
     @Override
     @Transactional
-    public boolean collaboratesOnAtLeastOneProject(String userName,
-            String collaboratorRole) throws QuadrigaStorageException {
-        return accessManager.getNrOfProjectsCollaboratingOn(userName,
-                collaboratorRole) > 0;
+    public boolean collaboratesOnAtLeastOneProject(String userName, String collaboratorRole)
+            throws QuadrigaStorageException {
+        return accessManager.getNrOfProjectsCollaboratingOn(userName, collaboratorRole) > 0;
     }
 
     /**
-     * This method checks if the user is project owner
+     * This method checks if the user has the specified collaboratorRole on the
+     * project
      * 
      * @param userName
-     * @return boolean - TRUE if the user is project owner else FALSE
+     * @param collaboratorRole
+     * @param projectId
+     * @return boolean - TRUE if the user has specified collaboratorRole else
+     *         FALSE
      * @throws QuadrigaStorageException
      * @author kiranbatna
-     * @throws NoSuchRoleException 
+     * @throws NoSuchRoleException
+     */
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.accesschecks.IProjectSecurityChecker#
+     * isCollaborator(java.lang.String, java.lang.String, java.lang.String)
      */
     @Override
     @Transactional
-    public boolean isCollaborator(String userName, String collaboratorRole,
-            String projectId) throws QuadrigaStorageException, NoSuchRoleException {
-        IQuadrigaRole role = roleManager.getQuadrigaRoleById(IQuadrigaRoleManager.PROJECT_ROLES, RoleNames.ROLE_PROJ_COLLABORATOR_ADMIN);
-        
+    public boolean isCollaborator(String userName, String collaboratorRole, String projectId)
+            throws QuadrigaStorageException, NoSuchRoleException {
+        IQuadrigaRole role = roleManager.getQuadrigaRoleById(IQuadrigaRoleManager.PROJECT_ROLES, collaboratorRole);
+
```

Comment:
By using passed "collaboratorRole", this method is used generically to check whether a user has specified collaboratorRole